src/clj/rems/application/cleaner.clj

```diff
@@ -0,0 +1,57 @@
+(ns rems.application.cleaner
+  (:require [mount.core :as mount]
+            [clj-time.core :as time]
+            [clojure.test :refer :all]
+            [clojure.tools.logging :as log]
+            [rems.scheduler :as scheduler]
+            [rems.db.applications :as applications]
+            [rems.config :refer [env]]))
+
+(defn- is-expired-draft?
+  [application]
+  (let [expiration-threshold (:draft-application-expiration-threshold env)
+        last-activity (:application/last-activity application)
+        state (:application/state application)]
+    (and (= :application.state/draft state)
+         (time/before? last-activity (time/minus (time/now) (time/days expiration-threshold))))))
+
+(deftest test-is-expired-draft
+  (with-redefs [env {:draft-application-expiration-threshold 90}]
+    (testing "should identify expired draft application"
+      (are [expected input] (= expected (is-expired-draft? input))
```

Comment:
We have so far wanted to spell out each `are` as `is` because it gives a more accurate trace including ability to jump to the correct line. We may want to consider that here too.